NoteEditorComponent.py

```diff
@@ -18,17 +13,21 @@ def __init__(self, stepsequencer = None, matrix = None, control_surface = None):
 		self._note_cache = None
 		self._playhead = None
 
-		# metronome
+		# playback step indicator
 		self.display_metronome = True
 		self.metronome_color = "StepSequencer.NoteEditor.Metronome"
+		
+		# playback page indicator
+		self._current_page = -1
 
-		# Velocity colour map. this must remain of lengh 3.
-		self.velocity_map = [70, 90, 110]
-		self.velocity_color_map = ["StepSequencer.NoteEditor.Velocity0", "StepSequencer.NoteEditor.Velocity1", "StepSequencer.NoteEditor.Velocity2"]
+		# Velocity color map. this must remain of length 3. WHY???
+		self.velocity_map = [20, 50, 80, 105, 127]
+		self.velocity_color_map = [	"StepSequencer.NoteEditor.Velocity0", "StepSequencer.NoteEditor.Velocity1", "StepSequencer.NoteEditor.Velocity2", "StepSequencer.NoteEditor.Velocity3", "StepSequencer.NoteEditor.Velocity4"]
```

Comment:
3, because it was the number of brightness levels the mk1 can display in one colour.
l'll have a look how it looks now :)